### DISCUSSION TITLE
Saving data in JSON format?

### DISCUSSION THREAD

**Discussion by mupeh** [Forceu/barcodebuddy#152] [Q&A]
Hi, I just found this repository and it looks great. 

I was looking for something like this with the purpose of data collection in two ways. 
- Keeping track of the money I spend. 
- Keeping track of the amount of sugars, fats, calories I consume. 

I haven't looked through the code yet but I was wondering how easy/difficult this would be to implement. (Getting a JSON file for items consumed in a day/ added in a day)

**Comment by Forceu** [comment]
I am afraid that would be out of scope. I guess your best bet would be to parse grocys stock logs with their API.
